_data/collection-index.yml

```
@@ -932,4 +932,8 @@
   maintainer: Victor M. Varela
   contact: https://github.com/vmvarela/devcontainer-features/issues
   repository: https://github.com/vmvarela/devcontainer-features
-  ociReference: ghcr.io/vmvarela/devcontainer-features
+- name: Crystal features
+  maintainer: Caesarovich
+  contact: https://github.com/Caesarovich/devcontainer-feature-crystal/issues
+  repository: https://github.com/Caesarovich/devcontainer-feature-crystal
+  ociReference: ghcr.io/Caesarovich/devcontainer-feature-crystal
```

```suggestion
  ociReference: ghcr.io/caesarovich/devcontainer-feature-crystal
```

Hi 👋

The ORAS spec requires the ociReference to be in lower case. Can we update here, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible.

---

Thanks for letting me know !